### ISSUE TITLE
Duplicate results in Accessibility Collector

### ISSUE THREAD

**Issue by wblachowski** [Cognifide/aet#438]
**Describe the bug**
Accessibility tab in report shows the same issue multiple times.

* Version used: `AET 3.1.0`

**To Reproduce**
1. Prepare a suite for https://www.softsoap.com/en-us/faq with accessibility checking (standard="WCAG2AAA", report-level="ERROR")
2. Run the suite
3. Go to accessibility tab on raport page
4. See duplicate results

**Expected behavior**
Each issue should be listed only once.

**Screenshots**
![bug](https://user-images.githubusercontent.com/20361302/48836018-ea2a3980-ed81-11e8-8f78-91c6782a1711.PNG)

**Comment by wblachowski** [comment]
Turns out that those are elements that actually occur more than once on the page provided. The problem however remains as in the report they are listed with the same line and column number.
